docs/index.rst

```diff
@@ -40,6 +40,7 @@ User Guide
    intro
    quickstart
    config
+   guides
```

Comment:
I'm not familiar with Sphinx so I may have missed some other place the new page had to be registered, but this seemed to be sufficient when building the docs locally